presto-product-tests/bin/locations.sh

```diff
@@ -2,3 +2,16 @@ SCRIPT_DIR=${BASH_SOURCE%/*}
 PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_CONF_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker"
+
+# Yes, JDBC_DRIVER_JAR is intentionally set to a directory. 
+# This works better than leaving it empty or unset.
+# The open source Presto JDBC driver is included in the fat jar with the tests.
+JDBC_DRIVER_JAR="${PRODUCT_TESTS_ROOT}"
```

Comment:
Please move this to compose-commons.sh. The value of this variable is of no interest to other places that `source locations.sh`. It's only needed for filling the variables in compose files, so let's have it in `compose-commons`.